### DISCUSSION TITLE
Is there a way to email a list of rows that meet certain conditions once a day?

### DISCUSSION THREAD

**Discussion by aidenmitchell** [Budibase/budibase#5141] [Help]
I'm trying to do the following:
1. Once a day, get all rows in a certain table and for each row, evaluate whether a) Stock is less than another value in the row AND b) a bool is false in the row
2. Email that list

This is what I have so far:
![image](https://user-images.githubusercontent.com/30846409/160456693-7eebdce7-4a17-4599-a3bc-f108a2e98894.png)
![image](https://user-images.githubusercontent.com/30846409/160457006-6d48f356-de11-470b-ac1c-70fb64adabc1.png)

My issue is that I don't know how to filter for "Stock less than binding", because I can't seem to reference the row in the filter dialog.

Thanks for the help!

**Comment by melohagan** [comment]
Hi @aidenmitchell 

To access the rows from the query and filter on the Notify At column, you will need to add another query after the first. 
![Screenshot 2022-03-29 at 11 09 43](https://user-images.githubusercontent.com/101575380/160588513-7ed1445f-eec0-4a32-a53d-e92d5a3bcdaf.png)

The tricky part here will be accessing the *Notify At* column for a specific row.
